javaslang/src/main/java/javaslang/collection/Seq.java

```diff
@@ -22,7 +18,7 @@
  * Interface for immutable sequential data structures.
  * <p>
  * Basic operations:
- *
+ * <p>
```

Comment:
no - I had to remove these previously (when invoking doclint), the &lt;ul&gt; already introduces a break!